ovh/resource_ovh_ip_reverse.go

```diff
@@ -0,0 +1,154 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type OvhIpReverse struct {
+	IpReverse string `json:"ipReverse"`
+	Reverse   string `json:"reverse"`
+}
+
+func resourceOvhIpReverse() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOvhIpReverseCreate,
+		Read:   resourceOvhIpReverseRead,
+		Update: resourceOvhIpReverseUpdate,
+		Delete: resourceOvhIpReverseDelete,
+
+		Schema: map[string]*schema.Schema{
+			"ip": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"ipreverse": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
should be optional and default to the IP if ip is /32
otherwise raise an error.
it would better reflect the API